magento/magento2
# Terms & Condition validation: Checkout error message "The order wasn't placed. First, agree to terms and conditions, then try placing your order again."

Terms & Condition validation

### Description (*)
Checkout error message "The order wasn't placed. First, agree to terms and conditions, then try placing your order again."

### Fixed Issues (if relevant)

1. Fixes magento/magento2#31461

### Manual testing scenarios (*)

1. First login on frontend
2. You must have at least one terms and conditions configured
3. Add a product to the cart
4. Go to checkout
5. Go to second step (payments)
6. Before selecting a payment method, open coupon code to be able to enter a coupon code
7. Than select a payment method.
8. As soon as you selected the payment method, the error message is not shown now. If you try to place order without selected the terms and condition then validation message appear.

### Contribution checklist (*)
